vendor/licenses/git_submodule/nix-linguist.dep.yml

```
@@ -0,0 +1,34 @@
+---
+name: nix-linguist
+version: 83d5d25e1ad087ea8e899194357fc8bdd8da3ee2
+type: git_submodule
+homepage: https://github.com/sambacha/nix-linguist.git
+license: other
```

Reviewed the license and can confirm this is MIT. Feel free to change this:

```suggestion
license: mit
```

---

💯